ui/app/common/displaycontrols/observation/services/formRecordTreeBuildService.js

```diff
@@ -103,7 +103,14 @@ angular.module('bahmni.common.displaycontrol.observation')
                             if (formDetailsAsString) {
                                 var formDef = JSON.parse(formDetailsAsString);
                                 formDef.version = observationForm.version;
-                                forms.push(self.updateObservationsWithRecordTree(formDef, form));
+                                formDef.version = observationForm.version;
```

Comment:
@vinishad looks like a duplicate line. can you please check

Reply:
@binduak thank you. Resolved and pushed